src/algorithms/LICENSE

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
move to root with name `LICENSE.LGPL3`

Reply:
That would imply that everything is LGPL, which is not what I'm trying to merge here (though I think it is). This location makes it clear that algorithms is LGPL, which no one argues against.